Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM mhart/alpine-node
 WORKDIR /usr/src/app
 COPY package.json /usr/src/app/
 
+RUN apk --update add git build-base && \
+    apk del build-base && rm -fr /usr/share/ri
```

Comment:
The `rm /usr/share/ri` is also because of the `bundle` step in the middle. I think you just want to add `git` here and that's all.